common/grid-layout/layout-en.html

```diff
@@ -0,0 +1,758 @@
+---
+{
+	"title": "Layout",
+	"language": "en",
+	"altLangPage": "layout-fr.html",
+	"description": "Bootstrap and WET classes showcasing layout options such as flexbox, CSS grid, grid baseline (containers) accompanied by practical examples to test and observe different page layouts.",
+	"dateModified": "2023-08-08"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#flexbox">Flexbox</a></li>
+	<li><a href="#css-grid">CSS-grid</a></li>
+	<li><a href="#grid baseline">Grid baseline (containers)</a></li>
+</ul>
+
+<h3 id="flexbox">Flexbox</h3>
+
+<h4>WET-BOEW <code>.d-flex</code> class working example</h4>
```

Comment:
```suggestion
<h4><code>.d-flex</code> class working example</h4>
```

Remove the "WET-BOEW" as it don't directly support the example but relate more to how to maintain that CSS/features

Reply:
Fixed